### PR TITLE
Update debug-scheduler-determinism

### PR DESCRIPTION
**Pull request by haslinghuis**
- Keep in sync with firmware https://github.com/betaflight/betaflight/pull/13230

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by sugaarK — APPROVED** [review]

**Review by SteveCEvans — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
